### ISSUE TITLE
Update the debian installer for systemd

### ISSUE THREAD

**Issue by aronasorman** [learningequality/installers#271]
Looks like Raspbian recently switched to `systemd` -- now we have a too early of a timeout for the Raspberry Pi 1. Ubuntu is also slated to switch to systemd soon, so with these two issues I think we should migrate to a systemd script.

@MCGallaspy this is the issue we were debugging.

**Comment by aronasorman** [comment]
I doubt I'll get to this early this week, so I'm assigning @benjaoming for now.

**Comment by MCGallaspy** [comment]
We'll also need a manual workaround. See https://github.com/learningequality/ka-lite/issues/4664#issuecomment-156631710.

**Comment by benjaoming** [comment]
No solution is coming for 0.15.1, anybody wanna block the release? @aronasorman ?

**Comment by aronasorman** [comment]
Nah, no time for me. 0.16.

**Comment by MCGallaspy** [comment]
Is this extant?

**Comment by benjaoming** [comment]
Yes, I encountered this (or something like it) on the latest Raspbian Jessy.

**Comment by benjaoming** [comment]
Going to have a look at this!

**Comment by benjaoming** [comment]
I'm running a Raspbian Jessie and there are no issues with the current init.d script. Downgrading this as a non-blocker unless someone has an opinion otherwise.

**Comment by benjaoming** [comment]
Here's a pattern we can copy to have both init.d and systemd scripts alongside:

http://sources.debian.net/src/thinkfan/0.9.2-1/debian/rules/

Thanks to the debian-systemd IRC channel :)

**Comment by benjaoming** [comment]
One important thing though will be to ensure that `init-system-helpers` are installed on all target platforms.

**Comment by benjaoming** [comment]
Auto-generated ka-lite.service file by Raspbian's systemd-sysv-generator:

```
# Automatically generated by systemd-sysv-generator

[Unit]
SourcePath=/etc/init.d/ka-lite
Description=LSB: ka-lite daemon, a local Khan Academy content server
After=local-fs.target remote-fs.target network-online.target systemd-journald-dev-log.socket nss-lookup.target
Wants=network-online.target

[Service]
Type=forking
Restart=no
TimeoutSec=5min
IgnoreSIGPIPE=no
KillMode=process
GuessMainPID=no
RemainAfterExit=yes
ExecStart=/etc/init.d/ka-lite start
ExecStop=/etc/init.d/ka-lite stop
```

**Comment by benjaoming** [comment]
This systemd bug is in the Wild, meaning that vanilla Raspbian installations without updates suffer from it: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=825913

The only solution is to start distributing a systemd startup service as the above.

**Comment by benjaoming** [comment]
Fixed in #422 - released in 0.16.9-ubuntu3
